eo-parser/src/test/resources/org/eolang/parser/packs/float-up-same-attrs.yaml

```diff
@@ -0,0 +1,31 @@
+xsls:
+  - /org/eolang/parser/optimize/remove-refs.xsl
+  - /org/eolang/parser/optimize/abstracts-float-up.xsl
+  - /org/eolang/parser/optimize/remove-levels.xsl
+  - /org/eolang/parser/add-refs.xsl
+tests:
+  - /program[count(.//o[@base='build' and not(@ref)])=2]
+  - /program/objects[count(o[@original-name='build'])=2]
```

Comment:
@mximp You can add expectations about new name that both objects will have

Reply:
@volodya-lombrozo I believe it doesn't matter for this test case: we don't care about the names here.

Reply:
@volodya-lombrozo It tests particular fact about the output xml. The fact is "we create two objects with the same base and two objects with same original name". For this test it's enough. There is no point in going deeper (which names objects have or what order they are placed in etc.). Just minimum enough to reveal the issue under consideration.